.github/workflows/node.js.yml

```diff
@@ -0,0 +1,37 @@
+
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
```

Comment:
it should be configured to develop branch or any branch so that it doesn't depend on your branch